api/CwatchApi.php

```diff
@@ -161,7 +161,38 @@ public function addsite($params)
 
         return new ApiResponse($responseBody);
     }
+    /**
+     * add site for a scan malware
+     * @param array $params
```

Comment:
The description for this method doesn't appear to make much sense to me, and the parameter/return statements have no description